pkg/cosign/keys.go

```diff
@@ -104,3 +108,147 @@ func UnmarshalPEMToPublicKey(pemBytes []byte) (crypto.PublicKey, error) {
 
 	return nil, fmt.Errorf("unsupported public key type or format")
 }
+
+/*
+  Adapted from sigstore/cosign to include support for OPENSSH PRIVATE KEY
+*/
```

Comment:
Can you include a link to the code you adapted this from?

Reply:
Sure! Done.